randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -2207,8 +2207,26 @@ Extra - aabb: [743.8239, -217.36417, 29.29987, 834.11273, -144.96254, 81.37303]
   * Normal Door to Reflecting Pool Access/Door to Hall of the Elders
   * Extra - dock_index: 0
   * Extra - nonstandard: False
+  > Door to Crossway Access South
+      All of the following:
+          # https://youtu.be/C5DBRQ4gCIY
```

Comment:
It may be 720p 60fps, but the compression quality is terrible

You need to use different export settings to reduce the compression artifacts in your videos.

Also, Prime Practice Mod's input display is preferred.

Reply:
I was using pwootage's practice mod from May 16, 2019 from the resources channel, but the display was just a row of prompts on the top of the screen showing what buttons are being pressed, not an actual controller layout. That's why I went with a different one.

Reply:
https://github.com/MetroidPrimeModding/prime-practice-native/releases